doc/en/user/source/styling/workshop/ysld/point.rst

```diff
@@ -352,7 +363,7 @@ The **text** symbolizer with the **label** property are required to label Point
 Dynamic Styling
 ---------------
 
-#. We will quickly use **scalerank** to select content based on @scale selectors.
+#. We will quickly use **scalerank** to select content based on @scale filters.
```

Comment:
this is not done with a filter, perhaps use the word scale range?

Perhaps that is too pedantic.